packages/core/src/app/customer/checkoutSuggestion/BoltCheckoutSuggestion.tsx

```diff
@@ -52,6 +54,7 @@ const BoltCheckoutSuggestion: FunctionComponent<BoltCheckoutSuggestionProps> = (
     }
```

Comment:
Let's also add an event for when the bolt checkout modal opens automatically on L39

Reply:
what event did you have in mind specifically?

Reply:
also, it looks like line 39 is just setting `showSuggestion = true` so that the bolt checkout button can render

Reply:
Greg experienced this issue where he entered a Bolt email and it didn't trigger the logged in flow ([slack](https://boltpay.slack.com/archives/C01J8D375MK/p1658877239502819)). The hypothesis is that BigC never sent the /v1/account/exists request.

Do we have any events which can confirm this hypothesis?

Reply:
The `initializeCustomer` function calls the /v1/account/exists endpoint. `onInit` gets called if the /v1/account/exists request returns true